src/main/java/model/data/type/TextDataType.java

```diff
@@ -45,4 +45,4 @@ public String getMimeType() {
 		return mimeType;
```

Comment:
Can you annotate all the fields with descriptions for each of the DataTypes?  This datatype is used by Piazza users who are creating User Services..  Do not make them hidden from the model..